packages/pages/src/common/src/project/structure.ts

```diff
@@ -244,6 +244,27 @@ export class ProjectStructure {
     return [new Path(templatesRoot)];
   };
 
+  /**
+   * @returns the list of of src/modules, taking scope into account. If a scope is defined then
+   * both the scoped and non-scoped module paths are returned.
+   */
+  getModulePaths = () => {
```

Comment:
Alexis [wrote](https://github.com/yext/pages/pull/494/files#diff-9065153815b3fabd28e0778d526a6430d8985add32f659f18d19293b9d316901R278) one too but yours is more correct. When you merge take yours instead and update her code that was using her version.

Reply:
Ack for now

Her version returns just one path while mine can return multiple. I will update hers in another PR.